### ISSUE TITLE
Label does not render on Android.

### ISSUE THREAD

**Issue by AndrePech** [react-native-community/react-native-tab-view#565]
Label does not render on Android.

Labels rendering like on IOS 

### Code sample
I just copied the Example

### Screenshots (if applicable)
![screenshot_20180719-184321_appvendas](https://user-images.githubusercontent.com/4999683/42971725-b3dbca16-8b83-11e8-9583-a383e6ffd06d.jpg)

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | ANDROID 8
| react-native          | 

| react-native-tab-view |
| node                  |
| npm or yarn           |

**Comment by AndrePech** [comment]
It works on DEV MODE

**Comment by C0dekid** [comment]
You need to add `labelStyle` to your `TabBar` component. For example:

```js
labelStyle={{ color: '#FFFFFF', fontWeight: 'bold', fontSize: 14 }}
```

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
